pkg/ccl/changefeedccl/changefeed_dist_test.go

```diff
@@ -480,6 +481,83 @@ func (rdt *rangeDistributionTester) balancedDistributionUpperBound(numNodes int)
 	return int(math.Ceil((1 + rebalanceThreshold.Get(&rdt.lastNode.ClusterSettings().SV) + 0.1) * 64 / float64(numNodes)))
 }
 
+// Additional helper function
+func anyGreaterThanThirty(counts []int) bool {
+	for _, count := range counts {
+		if count > 30 {
+			return true
+		}
+	}
+	return false
+}
+
+type strategyOpt struct {
+	set bool
+	val changefeedbase.ChangefeedRangeDistributionStrategy
+}
+
+func (o strategyOpt) String() string {
+	if !o.set {
+		return "unspecified"
+	}
+	return string(o.val)
+}
+
+func TestChangefeedWithDistributionStrategyOptions(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	skip.UnderShort(t)
+	skip.UnderDuress(t)
+	skip.UnderStress(t)
+
+	noLocality := func(i int) []roachpb.Tier {
+		return nil
+	}
+	// The cluster setting is metamorphicly randomized for testing, so can't leave unspecified.
+	clusterOptions := []strategyOpt{
+		{set: true, val: changefeedbase.ChangefeedRangeDistributionStrategyDefault},
+		{set: true, val: changefeedbase.ChangefeedRangeDistributionStrategyBalancedSimple},
+	}
+	testutils.RunValues(t, "cluster options", clusterOptions, func(t *testing.T, clusterOption strategyOpt) {
```

Comment:
nit: for both `testutils.RunValues` calls, use the singular form for the name (i.e. "option") and inline the slices